### DISCUSSION TITLE
Suggestions for detecting birds and squirrels?

### DISCUSSION THREAD

**Discussion by mmcc-xx** [blakeblackshear/frigate#5992] [Ask A Question]
I have a camera pointed at a bird feeder and I'd like to detect birds (so I can monitor what sorts of birds are visiting) and squirrels (so I can somehow dissuade them from visiting).

The model built into Frigate knows about "bird" and does a good job of detecting them. Sadly it does not know about "squirrel".

From what I understand, I could train a new model on new images of birds a squirrels, but I'd lose the built in model's bird detecting ability. It would probably take quite a bit of training to replicate that level of bird detecting ability.

Is there some way to add squirrel detection without losing the existing object types? Perhaps I could set up two cameras with the same feed, and use a custom squirrel-specific model with one of them?

**Comment by NickM-27** [comment]
if you are looking to train a model, you can use knowledge transfer https://coral.ai/docs/edgetpu/retrain-detection/

**Reply by mmcc-xx** [reply]
The problem with that approach is that you lose the original set of detectable objects, which I'm hoping to avoid. Thanks!

**Reply by NickM-27** [reply]
I don't believe that is the case, otherwise "transfer learning" would be an incorrect name since the current model data would not be transferred

![Screen Shot 2023-04-11 at 07 13 26 AM](https://user-images.githubusercontent.com/14866235/231173757-ccdfb04c-c0d1-441c-9b1a-4c47511151de.png)

**Comment by mdisabato** [comment]
I added "mouse". That gets the chipmunks and likely sees the squirrels as oversize mice.

**Reply by blakeblackshear** [reply]
Mouse in the coco dataset is a computer mouse, not the animal.

**Reply by mdisabato** [reply]
That officially puts me in the "DUH" category! And here I thought they were just too small for the camera. OMG... too funny!!!!

*modifies his config file*
